src/main/java/helper/Parser.java

```diff
@@ -0,0 +1,44 @@
+package helper;
+
+import helper.command.*;
+
+/**
+ * Parsing class
+ */
+public class Parser {
+    /**
+     * Parses the user input for duke
```

Comment:
In the coding standard, there is one line:

> In method header comments, the first sentence should start in the form Returns ..., Sends ..., Adds ... (not Return or Returnning etc.)

Maybe you want to change every javadoc starts with verb?